Update SSPR data requirement- phone #s

cloudhandler opened the pull request:
Update from customer feedback, there needs to be a space between the country code and the 10 digit phone number in order for the phone field to sync for sspr registration.

azurecla commented:
Hi __@daviburg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

davidwrede commented:
@daviburg, thanks for the quick fix!

cloudhandler committed:
Update SSPR data requirement- phone #s

Update from customer feedback, there needs to be a space between the country code and the 10 digit phone number in order for the phone field to sync for sspr registration.

cloudhandler committed:
Update active-directory-passwords-data.md

MicrosoftGuyJFlo reviewed (approved):
Looks great! Thanks @cloudhandler.
